src/SourceBuild/content/test/Microsoft.DotNet.Tests/Directory.Build.props

```diff
@@ -0,0 +1,28 @@
+<Project>
```

Comment:
This whole file isn't required (since the IsTestProject property doesn't need to be set explicitly for this test project. That was the only reason why we needed a D.B.props file). Please move all this stuff into the project file.

Reply:
Fixed with https://github.com/dotnet/sdk/pull/44649/commits/04209d39c78c94a17e5919c59c84b79be05d9be8

Reply:
Actually fixed with https://github.com/dotnet/sdk/pull/44649/commits/6337002902416768d8776d865625b83fd6d75d79